src/debugpy/_vendored/pydevd/_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -624,10 +624,10 @@ def _get_func_code_info(code_obj, frame_or_depth) -> FuncCodeInfo:
             frame = frame_or_depth
         assert frame.f_code is code_obj
 
-    func_code_info.filtered_out_force_checked = py_db.apply_files_filter(frame, func_code_info.abs_path_filename, py_db.get_use_libraries_filter())
+    func_code_info.filtered_out_force_checked = py_db.apply_files_filter(frame, func_code_info.abs_path_filename, True)
 
     if py_db.is_files_filter_enabled:
-        func_code_info.always_filtered_out = func_code_info.filtered_out_force_checked
+        func_code_info.always_filtered_out = py_db.apply_files_filter(frame, func_code_info.abs_path_filename, False)
```

Comment:
always_filtered_out needs to recheck that the file is actually always filtered out. justMyCode would have to be true or it would have to match the exclusion filters if it's not in the user's project.